src/Maestro/Client/src/MaestroApiOptions.cs

```diff
@@ -22,6 +22,7 @@ public partial class MaestroApiOptions
 
         public const string StagingMaestroUri = "https://maestro.int-dot.net/";
         public const string OldPcsStagingUri = "https://maestro-int.westus2.cloudapp.azure.com/";
+        public const string PcsProdUri = "https://product-construction-prod.wittysky-0c79e3cc.westus2.azurecontainerapps.io";
```

Comment:
The others have a trailing slash

```suggestion
        public const string PcsProdUri = "https://product-construction-prod.wittysky-0c79e3cc.westus2.azurecontainerapps.io/";
```